### ISSUE TITLE
focus on connected links and nodes on hover

### ISSUE THREAD

**Issue by cjyetman** [christophergandrud/networkD3#168]
I have a development branch for a 'focus on connected links and nodes on hover' feature here: https://github.com/cjyetman/networkD3/tree/focus_on_hover

to test, one can install with...
```r
devtools::install_github('cjyetman/networkD3', ref = 'focus_on_hover')
```

This was mentioned [here](https://github.com/christophergandrud/networkD3/issues/99#issuecomment-280406495), but I gave it its own issue and branch because it's not necessarily linked to the arrowhead feature.

I only made changes to `forceNetwork()`, not `simpleNetwork()` (I'm going to propose in another issue to make `simpleNetwork()` an alias of `forceNetwork()` rather than maintaining two separate but very similar functions). In my testing, it seems to be working as expected.

Two important questions...
- should there be an option/argument that can turn this behavior off and on, and if so, what should the default be?
- if the default opacity is set pretty low, the difference you will see (currently `options.opactiy / 4`) might not be so significant... is that something we need to work around, or is it up to the user to realize that setting a very low default opacity will weaken this effect?

**Comment by cjyetman** [comment]
#103 is a similar request, though it suggests showing related nodes and links on click as opposed to on hover

**Comment by christophergandrud** [comment]
This is awesome. I really like it.

I think this should be the default behaviour and we don't need an argument for it. I wouldn't worry about the opacity issue. That seems like something people will figure out as you mention.

I know it's a separate issue, but the idea of having `simpleNetwork` as a alias of `forceNetwork` makes a lot of sense to me.

**Comment by cjyetman** [comment]
@christophergandrud should I take your comment "I think this should be the default behaviour and we don't need an argument for it" to mean that this should be not only the default behavior, but always on and the only behavior? If so, this could be wrapped up and merged quickly.

**Comment by christophergandrud** [comment]
Yep. Please do.

**Comment by cjyetman** [comment]
this is resolved by PR #178
